scripts/getDependencies.xml

```diff
@@ -22,6 +22,94 @@
 
 	<!-- set default LIB property to all -->
 	<property name="LIB" value="all" />
+   <target name="getJtregVersion">
+		<condition property="jtregTar" value="jtreg_5_1_b01">
+			<matches pattern="^([89]|1[02-6])$" string="${JDK_VERSION}"/>
+		</condition>
+
+		<condition property="jtregTar" value="jtreg_6_1">
+			<equals arg1="${JDK_VERSION}" arg2="11"/>
+		</condition>
+
+		<condition property="jtregTar" value="jtreg_7_3_1_1">
+			<matches pattern="^(17|2[12])$" string="${JDK_VERSION}"/>
+		</condition>
+
+		<fail message="Unsupported JDK version: ${JDK_VERSION}">
+			<condition>
+				<not>
+					<or>
+						<matches pattern="^([89]|1[02-6])$" string="${JDK_VERSION}"/>
+						<equals arg1="${JDK_VERSION}" arg2="11"/>
+						<matches pattern="^(17|2[12])$" string="${JDK_VERSION}"/>
+					</or>
+				</not>
+			</condition>
+		</fail>
```

Comment:
Please use the original logic https://github.com/adoptium/aqa-tests/blob/master/openjdk/build.xml#L44-L67